containerm/ctr/ctrd_client_internal.go

```diff
@@ -137,10 +155,50 @@ func (ctrdClient *containerdClient) pullImage(ctx context.Context, imageInfo typ
 		if checkErr := ctrdClient.decMgr.CheckAuthorization(ctx, ctrdImage, dc); checkErr != nil {
 			return nil, checkErr
 		}
+		if verifyErr := ctrdClient.verifyImage(ctx, ctrdImage, true, vk); verifyErr != nil {
+			return nil, verifyErr
+		}
 	}
 	return ctrdImage, err
 }
 
+var skipVerificationMessage = "signature verification is skipped for image = %s, %s"
```

Comment:
```suggestion
const skipVerificationMessage = "signature verification is skipped for image = %s, %s"
Write
```
I think this can also be moved in the `verifyImage(...)` method if it is not used anywhere else.